.gitignore

```
@@ -0,0 +1,10 @@
+# dependencies
+/node_modules
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+.idea
+
+TODO: terminate files with new line per unix standard
```

The sensible choice of terms here would be end of line (EOL) given it is intended here to represent the _end_ of a line, not the beginning of an always empty line.

```suggestion
TODO: terminate files with end of line per unix standard
```